pandas/io/tests/test_gbq.py

```diff
@@ -279,6 +273,15 @@ def test_should_be_able_to_get_results_from_query(self):
 class GBQUnitTests(tm.TestCase):
     def setUp(self):
         test_requirements()
+        self.sut_pub = gbq.GbqConnector('bigquery-public-data')
```

Comment:
The other test classes have `self.sut` as the name for the `GbqConnector` instance that connects to the pandas GBQ project. I named the connector to the public data `self.sut_pub` to indicate that it connects to the public data. I changed it to `self.sut_public`. I don't know what "sut" is... maybe it refers to "system under test"?